### ISSUE TITLE
Pineapple

### ISSUE THREAD

**Issue by westnordost** [rugk/crops-parser#31]
I think the current picture for the pineapple is not so good. It should be a photo slightly from above, so that not a house is in the background but the plant itself.

**Comment by rugk** [comment]
But it looks really great. Or we just crop it again a bit more…

**Comment by westnordost** [comment]
I am only giving you feedback, in the end it will be your quest, I only helped you. In other words, I leave the decision up to you.

**Comment by rugk** [comment]
I've looked for other pictures again, but only found silly pictures of ananas [being put somewhere](https://pixabay.com/de/ananas-sommer-sommer-stimmung-obst-1602349/) [in](https://pixabay.com/de/ananas-felsen-obst-entspannung-1149384/) [places](https://pixabay.com/de/ananas-sonnenuntergang-obst-strand-1621345/) [or](https://pixabay.com/de/strand-k%C3%BCste-obst-ozean-ananas-1834329/) [so](https://pixabay.com/de/strand-k%C3%BCste-obst-insel-ozean-1850818/)…

And the other :pineapple: just [were not](https://ccsearch.creativecommons.org/image/detail/6rPvfemb5mM1TKXS3ESiHg==) that nice and ripe as the current pic…
So basically I just cropped the current image slightly.
